draft-xyz-rats-psa-endorsements.md

```diff
@@ -131,94 +140,149 @@ firmware measurement associated with Implementation ID
 ~~~
 {: #ex-reference-value title="Example Reference Value"}
 
-### Firmware Updates and Patches
-{: #sec-fw-evo}
+## Identity
 
```

Comment:
the reference anchor must be adjacent to the element it refers to (in this case the section's title)
```suggestion
```

Reply:
This was giving a warning that Headings should be surrounded by blank lines!